plugins/modules/ipastageuser.py

```diff
@@ -0,0 +1,1361 @@
+      first:
+        description: The first name
+        required: false
+        aliases: ["givenname"]
+      last:
+        description: The last name
+        required: false
+        aliases: ["sn"]
+      fullname:
+        description: The full name
+        required: false
+        aliases: ["cn"]
+      displayname:
+        description: The display name
+        required: false
+      initials:
+        description: Initials
+        required: false
+      homedir:
+        description: The home directory
+        required: false
+      shell:
+        description: The login shell
+        required: false
+        aliases: ["loginshell"]
+      email:
+        description: List of email addresses
+        required: false
+      principal:
+        description: The kerberos principal
+        required: false
+        aliases: ["principalname", "krbprincipalname"]
+      principalexpiration:
+        description:
+        - The kerberos principal expiration date
+        - (possible formats: YYYYMMddHHmmssZ, YYYY-MM-ddTHH:mm:ssZ,
+        - YYYY-MM-ddTHH:mmZ, YYYY-MM-ddZ, YYYY-MM-dd HH:mm:ssZ,
+        - YYYY-MM-dd HH:mmZ) The trailing 'Z' can be skipped.
+        required: false
+        aliases: ["krbprincipalexpiration"]
+      passwordexpiration:
+        description:
+        - The kerberos password expiration date (FreeIPA-4.7+)
+        - (possible formats: YYYYMMddHHmmssZ, YYYY-MM-ddTHH:mm:ssZ,
+        - YYYY-MM-ddTHH:mmZ, YYYY-MM-ddZ, YYYY-MM-dd HH:mm:ssZ,
+        - YYYY-MM-dd HH:mmZ) The trailing 'Z' can be skipped.
+        - Only usable with IPA versions 4.7 and up.
+        required: false
+        aliases: ["krbpasswordexpiration"]
+      password:
+        description: The user password
+        required: false
+      random:
+        description: Generate a random user password
+        required: false
+        type: bool
+      uid:
+        description: The UID
+        required: false
+        aliases: ["uidnumber"]
+      gid:
+        description: The GID
+        required: false
+        aliases: ["gidnumber"]
+      city:
+        description: City
+        required: false
+      userstate:
+        description: State/Province
+        required: false
+        aliases: ["st"]
+      postalcode:
+        description: Postalcode/ZIP
+        required: false
+        aliases: ["zip"]
+      phone:
+        description: List of telephone numbers
+        required: false
+        aliases: ["telephonenumber"]
+      mobile:
+        description: List of mobile telephone numbers
+        required: false
+      pager:
+        description: List of pager numbers
+        required: false
+      fax:
+        description: List of fax numbers
+        required: false
+        aliases: ["facsimiletelephonenumber"]
+      orgunit:
+        description: Org. Unit
+        required: false
+      title:
+        description: The job title
+        required: false
+      manager:
+        description: List of managers
```

Comment:
Please, add that managers must be an active user.

Reply:
ack @rjeffman.  I will be working on this later this evening.  Thanks for the response.